client/src/document/index.tsx

```diff
@@ -249,30 +249,32 @@ function RenderDocumentBody({ doc }) {
 
 function LoadingError({ error }) {
   return (
-    <div className="page-content-container loading-error">
-      <h3>Loading Error</h3>
-      {error instanceof window.Response ? (
+    <div className="standard-page">
+      <div className="page-content-container loading-error">
```

Comment:
I would add the content id attribute to this element so the `a11y-nav` links to it.

```suggestion
      <div id="content" className="page-content-container loading-error">
```